transaction-flow.md

```diff
@@ -154,7 +154,7 @@ When a sender is created you will receive a response which contains the senders
 Notes:
 
 * The sender's phone number is composed of two parts, the `phone_country` (in ISO 2-letter format), and the `phone_number`. The phone number should be specified without the international prefix.
-* The `documents` should contain all documents necessary to KYC the sender. If you already do KYC on your system, then please contact us and we can waive this requirement from you. In this case you can send us an empty list of documents: `[]`. In case you don't do KYC on your site, then you will need to send us documents, for this please see the [API reference documentation](https://api.bitpesa.co/documentation#documents)
+* The `documents` should contain all documents necessary to KYC the sender. If you already do KYC on your system, then please contact us and we can waive this requirement from you. In this case you can send us an empty list of documents: `[]`. In case you don't do KYC on your site, then you will need to send us documents, for this please see the [API reference documentation](https://api.bitpesa.co/documentation#documents). In case you already do KYC on your system and we waive the sender documents requirements, then all of the senders you create in the system will be immediately set to the `approved` state and you won't need to wait for them to get approved.
```

Comment:
I know i just approved this, but it might make more sense to put the "auto-approved" note immediately after the initial sentence about waiving KYC, rather than splitting it to after a sentence about non-waived KYC.

Reply:
Good point, reorganized the section a bit